core/src/test/java/org/apache/druid/timeline/DataSegmentTest.java

```diff
@@ -51,7 +51,7 @@
 public class DataSegmentTest
 {
   private static final ObjectMapper mapper = new TestObjectMapper();
-  private static final int TEST_VERSION = 0x7;
+  private static final int TEST_VERSION = 0x9;
```

Comment:
Why this change?

Reply:
This represents the segment version. Its value doesn't affect to anything, but I think it's better to use a more realistic one since version 7 no more exists.